_dashboards/reporting.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: Reporting
+title: Creating reports from the Dashboards UI
 nav_order: 70
 ---
 
 
-# Reporting
+# Creating reports from the Dashboards UI
```

Comment:
This looks good to distinguish it from the new "Creating reports with the Reporting CLI" topic that is in-progress. 
nit . .  .maybe say "with" the Dashboards?

Reply:
Accepted.

Reply:
Revised